internal/network/command_private_link_attachment.go

```diff
@@ -0,0 +1,94 @@
+package network
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	networkingprivatelinkv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking-privatelink/v1"
+
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type privateLinkAttachmentOut struct {
+	Id                    string `human:"ID" serialized:"id"`
+	Name                  string `human:"Name" serialized:"name"`
+	Cloud                 string `human:"Cloud" serialized:"cloud"`
+	Region                string `human:"Region" serialized:"region"`
+	AwsVpcEndpointService string `human:"AWS VPC Endpoint Service,omitempty" serialized:"aws_vpc_endpoint_service,omitempty"`
+	Phase                 string `human:"Phase" serialized:"phase"`
+}
+
+func (c *command) newPrivateLinkAttachmentCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "attachment",
+		Short: "Manage private link attachments.",
+		Args:  cobra.NoArgs,
+	}
+
+	cmd.AddCommand(c.newPrivateLinkAttachmentDescribeCommand())
+	cmd.AddCommand(c.newPrivateLinkAttachmentListCommand())
+
+	return cmd
+}
+
+func (c *command) getPrivateLinkAttachments() ([]networkingprivatelinkv1.NetworkingV1PrivateLinkAttachment, error) {
```

Comment:
In other places we normally put getter in the package, and pass env id as an arg to call it.

Reply:
Thank you for taking a look!

Which package are you referring to? I am following this pattern from https://github.com/confluentinc/cli/blob/4fccbf2e3fb7bc4593abbacdc2bc836f6d9ced61/internal/pipeline/command.go#L136, and we have been using this pattern for other list commands.

Reply:
i was referring to putting the getter methon into `pkg/ccloudv2/(e.g.)network`, but if you can find other places of using the method this way i think it's fine then